stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewHolderFactory.kt

```diff
@@ -38,6 +33,8 @@ public open class MessageListItemViewHolderFactory {
     protected lateinit var attachmentViewFactory: AttachmentViewFactory
         private set
 
+    private lateinit var style: MessageListItemStyle
```

Comment:
This could be `internal` and then we wouldn't need a setter function for it

Reply:
Let it leave as is. I'm going to expose it as protected in next PR with AttachmentViewFactory improvement

Reply:
Are we making the `MessageListItemStyle` class itself public then?

Reply:
Yes, I'll have to do that. I see we had the same in old sample app